tests/beaker_tests/cisco_interface/routedintf_provider_nondefaults.rb

```diff
@@ -152,6 +156,10 @@
                                  /speed 100/,
                                  /duplex full/,
                                  /vrf member test1/,
+                                 /ip access-group v4acl1 in/,
```

Comment:
We need to remove them.  This will not scale across multiple platforms but using the `puppet resource` command can be used on any platform that supports puppet.